src/Form.php

```diff
@@ -616,18 +616,13 @@ public function ajaxSubmit()
                     return new jsExpression('console.log([])', ['Form submission is not handled']);
                 }
             } catch (\atk4\data\ValidationException $val) {
+                ob_get_clean(); // not close output buffer on exceptions
```

Comment:
output buffer
if the code in try/catch not raise exception it will exit from return $response with buffer cleaned.
if an exception is raised the old finally will not be executed leaving an output buffer open.
i remove the finally ( even more because the other catches has a return.
i add a ob_get_clean which can be ob_clean when exception is raised and changed ob_get_contents to ob_get_clean directly in try.